js/defs.bzl

```diff
@@ -7,19 +7,13 @@ load(
     _js_test = "js_test",
 )
 load(
-    "//js/private:js_package.bzl",
```

Comment:
include the typical `load` statement in the module doc at the top?

Reply:
✅